trimesh/collision.py

```diff
@@ -8,7 +8,6 @@
 try:
     import fcl  # pip install python-fcl
 except BaseException:
-    log.warning('No FCL -- collision checking will not work')
```

Comment:
This is because it's too noisy, and make it consistent to the import of `PIL.Image` or other libraries.